index.rst

```diff
@@ -565,47 +542,41 @@ Il *focus* sui processi consente di:
    coinvolta in attività diverse, come la progettazione dello schema
-   normativo, la regolamentazione delle modalità di accesso, la
-   gestione finanziaria dell'erogazione o le attività di
-   comunicazione all'esterno. La finalità è, tuttavia, comune, e
-   consiste nell'erogazione efficace ed efficiente degli incentivi.
-   L'analisi per processi consente di individuare indicatori di
-   *performance* trasversali legati all'utente finale e di
-   individuare, attraverso le attività, i contributi più specifici
-   delle singole unità organizzative;
-
+   normativo, la regolamentazione delle modalità di accesso, la gestione
+   finanziaria dell'erogazione o le attività di comunicazione
+   all'esterno. La finalità è, tuttavia, comune, e consiste
+   nell'erogazione efficace ed efficiente degli incentivi. L'analisi per
+   processi consente di individuare indicatori di *performance*
+   trasversali legati all'utente finale e di individuare, attraverso le
+   attività, i contributi più specifici delle singole unità
+   organizzative;
 -  agevolare la corretta individuazione degli *output* prodotti
-   dall'amministrazione, con particolare riferimento ai servizi
-   erogati e alla conseguente identificazione degli utenti (interni
-   o esterni) a cui questi ultimi sono rivolti, rendendo più
-   immediato il collegamento tra misurazione e valutazione della
-   *performance* organizzativa e gli esiti delle indagini sul
-   livello di soddisfazione di cittadini e utenti rispetto ai
-   servizi erogati;
-
+   dall'amministrazione, con particolare riferimento ai servizi erogati
+   e alla conseguente identificazione degli utenti (interni o esterni) a
+   cui questi ultimi sono rivolti, rendendo più immediato il
+   collegamento tra misurazione e valutazione della *performance*
+   organizzativa e gli esiti delle indagini sul livello di soddisfazione
+   di cittadini e utenti rispetto ai servizi erogati;
 -  favorire una più corretta pianificazione delle “risorse” necessarie
-   per realizzare gli *output*. Con particolare riferimento alle
-   risorse umane questo approccio risulta anche coerente con le
-   recenti modifiche normative in materia di pianificazione dei
-   fabbisogni di profili professionali che non devono essere più
-   legati alla dotazione organica;
-
+   per realizzare gli *output*. Con particolare riferimento alle risorse
+   umane questo approccio risulta anche coerente con le recenti
+   modifiche normative in materia di pianificazione dei fabbisogni di
+   profili professionali che non devono essere più legati alla dotazione
+   organica;
 -  incentivare la collaborazione tra unità organizzative.
 
 Infine è utile riprendere il concetto di **progetto** nel contesto di
 queste linee guida. Esso è un **insieme di attività finalizzato a
-realizzare un *output* ben preciso in un determinato intervallo di
+realizzare un output ben preciso in un determinato intervallo di
```

Comment:
@danse 
È interessante che qui abbia tolto gli asterischi... forse perché nello stesso paragrafo c'è un altro grassetto?

Reply:
questi asterischi sono all'interno di un blocco in grassetto cui Github non applica lo stile ... probabilmente non usano pandoc come parser ;)

Reply:
Hai ragione, non avevo visto i due asterischi prima di "insieme"